### PR TITLE
Mark service.version as stable.

### PR DESCRIPTION
**Pull request by carlosalberto**
Reopening this from the related Specification PR. 

cc @jsuereth

### CODE REVIEW

**Review by jsuereth — APPROVED** [review]

**Review by lmolkova — APPROVED** [review]

**Review by joaopgrassi — APPROVED** [review]

**Review by reyang — APPROVED** [review]

File: specification/resource/semantic_conventions/README.md [hunk 1]
```diff
@@ -76,6 +77,7 @@ as specified in the [Resource SDK specification](https://github.com/open-telemet
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
 | `service.name` | string | Logical name of the service. [1] | `shoppingcart` | Required |
+| `service.version` | string | The version string of the service API or implementation. | `2.0.0` | Recommended |
```
Comment (arminru) [inline specification/resource/semantic_conventions/README.md:80]: We could consider adding a second example so readers don't mistakenly assume it's always a semver string.

```suggestion
| `service.version` | string | The version string of the service API or implementation. | `2.0.0`, `a01dbef8a` | Recommended |
```

**Review by arminru — APPROVED** [review]

**Comment by carlosalberto** [comment]
FYI, our latest change clarifies that version can technically be any string, as defined by the owner, etc.

**Review by Oberon00 — APPROVED** [review]

File: semantic_conventions/resource/service.yaml [hunk 2]
```diff
@@ -16,3 +16,8 @@ groups:
           with [`process.executable.name`](process.md#process), e.g. `unknown_service:bash`.
           If `process.executable.name` is not available, the value MUST be set to `unknown_service`.
         examples: ["shoppingcart"]
+      - id: version
+        type: string
+        brief: >
+          The version string of the service API or implementation. The format is not defined by these conventions.
+        examples: ["2.0.0"]
```
[the remark this answers is not in the document]
Comment (arminru) [reply semantic_conventions/resource/service.yaml:23]: Good idea, thank you!

File: specification/resource/semantic_conventions/README.md [hunk 3]
```diff
@@ -76,6 +77,7 @@ as specified in the [Resource SDK specification](https://github.com/open-telemet
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
 | `service.name` | string | Logical name of the service. [1] | `shoppingcart` | Required |
+| `service.version` | string | The version string of the service API or implementation. The format is not defined by these conventions. | `2.0.0`, `a01dbef8a` | Recommended |
```
Comment (arminru) [inline specification/resource/semantic_conventions/README.md:80]:
```suggestion
| `service.version` | string | The version string of the service API or implementation. The format is not defined by these conventions. | `2.0.0`; `a01dbef8a` | Recommended |
```

**Review by AlexanderWert — APPROVED** [review]

**Pull request merged**